### ISSUE TITLE
Trailing stop market

### ISSUE THREAD

**Issue by iceryan354** [Altamoon/altamoon#8]
## Summary

Looks like a fantastic project. 
Is trailing stop buy/sell supported...? 
Signal buy, sell integration possible with plugin development? 

Cheera

**Comment by Letiliel** [comment]
Thanks!
Trailing stop is not supported at the moment and the project is going to be put under hibernation next month for an unknown duration, for life reasons.

> Signal buy, sell integration possible with plugin development?
I believe basically anything can be done with plugins. Maybe @finom can confirm?

**Comment by iceryan354** [comment]
Cheers thanks. Best of health and luck with life stuff 👌

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by finom** [comment]
> I believe basically anything can be done with plugins. Maybe @finom can confirm?

Yes but unfortunately there is no good documentation for it yet.
